### PR TITLE
Add veranstaltungstickets info

### LINKED ISSUE #35: add DB veranstaltungsticket info
link is https://www.veranstaltungsticket-bahn.de/?event=20421&language=en

### PR DESCRIPTION
**Pull request by thibaultamartin**
Closes #35

### CODE REVIEW

**Comment by thibaultamartin** [comment]
@HarHarLinks eyes welcome here too

File: templates/attend.html [hunk 1]
```diff
@@ -114,6 +114,10 @@ <h2>Central Station</h2>
                         stop at Fuldastraße. Take in to consideration that the line <span
                             class="public-transport public-transport-bus">M41</span> is often delayed due to traffic.
                     </p>
+
+                    <p>You can get cheaper tickets by <a
+                            href="https://www.veranstaltungsticket-bahn.de/?event=20421&language=en">buying
+                            "veranstaltungstickets"</a>. Those are only valid with a Matrix Conference ticket!</p>
```
Comment (HarHarLinks) [inline templates/attend.html:120]:
```suggestion
                    <p>The Matrix Conference is eligible for the <a
                            href="https://www.veranstaltungsticket-bahn.de/?event=20421&language=en">buying
                            "Veranstaltungsticket"</a> special offer by Deutsche Bahn for holders of 
                            Matrix Conference tickets!</p>
```

**Review by HarHarLinks — CHANGES_REQUESTED** [review]
i would move the paragraph to before the local bus part. first train, then bus.

File: templates/attend.html [hunk 2]
```diff
@@ -114,6 +114,11 @@ <h2>Central Station</h2>
                         stop at Fuldastraße. Take in to consideration that the line <span
                             class="public-transport public-transport-bus">M41</span> is often delayed due to traffic.
                     </p>
+
+                    <p>The Matrix Conference is eligible for the <a
+                            href="https://www.veranstaltungsticket-bahn.de/?event=20421&language=en">buying
```
Comment (HarHarLinks) [inline templates/attend.html:119]:
```suggestion
                            href="https://www.veranstaltungsticket-bahn.de/?event=20421&language=en">
```

File: templates/attend.html [hunk 3]
```diff
@@ -114,6 +114,11 @@ <h2>Central Station</h2>
...
+                            "Veranstaltungsticket"</a> special offer by Deutsche Bahn for holders of
```
Comment (HarHarLinks) [inline templates/attend.html:120]:
```suggestion
                    <p>The Matrix Conference is eligible for the "<a
                            href="https://www.veranstaltungsticket-bahn.de/?event=20421&language=en">
                            Veranstaltungsticket</a>" special offer by Deutsche Bahn for holders of 
```

**Comment by HarHarLinks** [comment]
ehhh can you remove the extra space between `"` and Veranstaltungsticket? also I think we use different quotes across the website. some are fancier than others maybe originating from gdocs (did I say to get rid of google yet today?)

**Pull request merged**